tests/integration/tests/admin-areas/admin-areas.test.ts

```diff
@@ -0,0 +1,301 @@
+    adminLevel: AdminLevel.adminLevel3,
+    scenario: FloodsScenario.Trigger,
+    leadTime: LeadTime.day0,
+    eventName: 'G5075',
+    placeCodeParent: '21UGA003004',
+    featureCount: 2,
+    placeCodeRegex: /^(UG|21UGA)/, // REFACTOR: set to /^UG/ after data is fixed
+  },
+];
+
+export default function adminAreaTests() {
+  describe('admin areas', () => {
+    let accessToken: string;
+
+    beforeAll(async () => {
+      accessToken = await getAccessToken();
+    });
+
+    assertions.forEach(
+      ({
+        countryCodeISO3,
+        disasterType,
+        adminLevel,
+        scenario,
+        featureCount,
+        placeCodeRegex,
+        leadTime,
+        eventName,
+        placeCodeParent,
+      }) => {
+        let title = `${countryCodeISO3} / ${disasterType} / ${adminLevel} / ${scenario}`;
+        if (leadTime) {
+          title += ` / ${leadTime}`;
+        }
+        if (eventName) {
+          title += ` / ${eventName}`;
+        }
+        if (placeCodeParent) {
+          title += ` / ${placeCodeParent}`;
+        }
+        it(title, async () => {
+          // Arrange
+          await mock(
+            scenario,
+            disasterType,
+            countryCodeISO3,
+            null,
+            accessToken,
+          );
+
+          // Act
+          const adminAreas = await getAdminAreas(
+            countryCodeISO3,
+            disasterType,
+            adminLevel,
+            accessToken,
+            leadTime,
+            eventName,
+            placeCodeParent,
+          );
+
+          // Assert
+          expect(adminAreas.status).toBe(200);
+          expect(adminAreas.body.features.length).toBe(featureCount); // we expect a deterministic feature count from the mock data
+
+          const feature = adminAreas.body.features[0]; // test the first feature
+          expect(feature.geometry.type).toBe('MultiPolygon');
+          expect(feature.geometry.coordinates[0][0].length).toBeGreaterThan(0); // the coordinates array should not be empty
+          expect(feature.geometry.coordinates[0][0][0].length).toBe(2); // the coordinates should be in [longitude, latitude] format
+          expect(feature.properties.placeCode).toMatch(placeCodeRegex); // placeCode should match regex per country
+          if (eventName) {
+            // all admin areas in an event should have the same alert level
+            expect(
+              adminAreas.body.features.every(
+                ({ properties: { alertLevel } }) =>
+                  alertLevel === feature.properties.alertLevel,
+              ),
+            ).toBeTruthy(); // all features should have the same alert level
+          }
+          if (
+            countryCodeISO3 === 'MWI' &&
+            disasterType === DisasterType.FlashFloods
+          ) {
+            return;
+          }
+          // REFACTOR: find out why MWI flash-flood responds differently, fix it, and remove this conditional assertion
```

Comment:
I guess this is because of the event-areas right? This endpoint returns event-areas in this case instead of admin-areas. (in National View, so without eventName). If not something that can be made actionable for now in the tests, let's at least update the comment to reflect this insight.

Reply:
Updated https://github.com/rodekruis/IBF-system/pull/2121/commits/ea51c3f33875f36177f3592119eae61796558068

I still don't understand why event areas do not contain a `name / admin level / country code`.

Feel free to update the comment with any more information that will help.

@jannisvisser

Reply:
This sounds like this can be improved, probably as part of AB#32715. Making the response format (more) aligned with the admin-area response format. In fact: 
- Country code could just be added (although not used), 
- name is now 'eventAreaName', but it doesn't seem to be used, so could just be changed to 'name' it seems. 
- only adminLevel is a problem, because event-areas really do not have an adminLevel. But the other 2 could maybe be changed in this PR.

Reply:
Updated https://github.com/rodekruis/IBF-system/pull/2121/commits/2833a1c1f3a90f7e9fed6b1a2c2e3976cab6b9cc

- I aligned the respone format of event-areas (more) to that of admin-areas. But left the refactor comment, that more can be done.
- Refined the test exception to only do so if no eventName (as only in National View this is different) and removed the countryCode. As the code exception on this is on disaster-type currently as well, not on country.